yes.rs

```diff
@@ -1,19 +1,11 @@
 use std::env;
 
-#[allow(while_true)]
 fn main() {
     let args: Vec<String> = env::args().collect();
+    let args = &args[1..];
 
-    match args.len() {
-        1 => {
-            while true {
-                println!("y");
-            }
-        },
-        _ => {
-            while true {
-                println!("{}", args[1]);
-            }
-        },
+    let output = if args.is_empty() { "y".to_string() } else { args.join(" ") };
```

Comment:
Ahh, now I found the way you could do this which is pretty simple.

    let args = args[1..].join(" ");
    let output = if args.is_empty() { "y" } else { &args };